src/components/Modals/CreateNewDatabase/CreateDBModal.jsx

```diff
@@ -23,25 +14,18 @@ import InputsArea from "../SharedItems/InputsArea";
 import InputWrapper from "../SharedItems/InputWrapper";
 import CreateDBInputList from "./CreateDBInputList";
 import Button from "../../shared/Button";
-import Loading from "../../shared/Loading";
 
 import CONSTANT from "../../../constants/constant";
 
 function CreateDBModal() {
-  const queryClient = useQueryClient();
-  const navigate = useNavigate();
-  const { userId } = useAtomValue(userAtom);
-
+  const fetchNewDatabase = usePostDB();
   const [dbName, setdbName] = useState(null);
```

Comment:
클라이언트 로직 전반적으로 `DB`, `Db`, `db`가 혼용되고 있는 것 같아, 이후에 카멜 케이스로 통일하는게 어떨까요?
```suggestion
  const [dbName, setDbName] = useState(null);
```

마이너하다고 생각될 수도 있는데, 계속 눈에 밟히는 부분이라 건의드려봅니다. 백엔드 로직도 혼용되고 있는 부분이 있는걸로 기억해서.. 😭

Reply:
피드백 감사합니다! 마침 눈에 띌때마다 DB로 통일해주고 있는 상황인데 아직 남아있었네요.. 작업해서 commit하겠습니다. (database의 약자인것과 가독성을 고려하여 DB가 좋겠다고 판단하여 통일진행하에 있습니다.)